src/main/resources/rulesets/checkstyle/rules.xml

```diff
@@ -82,6 +82,12 @@
   <module name="org.openhab.tools.analysis.checkstyle.ImportExportedPackagesCheck">
     <property name="severity" value="warning" />
   </module>
+  
+  <module name="org.openhab.tools.analysis.checkstyle.ManifestPackageVersionCheck">
+     <property name="severity" value="warning" />
+     <property name="ignoreImportedPackages" value="org.junit.*,
```

Comment:
It is not explicitly written, but reading the guidelines left with this impression. Maybe we should ask for another opinion too.